pkgs/unified_analytics/lib/src/session.dart

```diff
@@ -49,7 +49,8 @@ class Session {
     if (now.difference(lastPingDateTime).inMinutes > kSessionDurationMinutes) {
       // Update the session file with the latest session id
       _sessionId = now.millisecondsSinceEpoch;
-      sessionFile.writeAsStringSync('{"session_id": $_sessionId}');
+      sessionFile.writeAsStringSync(
```

Comment:
can we create a method that does this write, so we make sure to always keep the multiple call sites in sync?